orchestra/model/configuration.py

```diff
@@ -177,7 +177,7 @@ def _compute_global_env(self):
         # TODO: the order of the variables stays the same even if the
         #  user overrides an environment variable from the config.
         #  This is convenient but we should think if it is really what we want.
```

Comment:
Weird indentation. This is a recurrent issue, please find a definitive solution.

Reply:
Are you referring to the fact that the line is not near the 120 column or that the second and subsequent lines are indented with two spaces?
I did both on purpose: I broke earlier because the whole message does not fit on a single line, so instead of having one long line and a very short one I preferred three similarly-long lines.
The over-indentation is because my IDE sees the three lines as one single TODO by over-indenting subsequent lines.
How would you like them to be formatted?

Reply:
We usually do it like this:
```python
# TODO: the order of the variables stays the same even if the user overrides an                                                                                                               
#       environment variable from the config.  This is convenient but we should                                                                                                               
#       think if it is really what we want.
```
Or don't indent the second line at all, if you prefer. But when I look at these comments it always feel like some weird business is going on. 

I'd avoid being smart, just to wrap whenever needed. I feel like `vim` and `emacs` have each some command to do something like this.

Reply:
Fixed all occurrences I could find by grepping "`#  `"